mlox_user.txt

```diff
@@ -3707,6 +3707,16 @@ Clean Royal Chargen.esp
 Mournhold LOD.esp
 Immersive Mournhold.esp
 
+[Order] ; (Ref: https://www.nexusmods.com/morrowind/mods/52601 )
+Karethys consort wizard.ESP
+Immersive Mournhold.esp
+
+[Note] ; (Ref: https://www.nexusmods.com/morrowind/mods/52601 ) (MasssiveJuice)
```

Comment:
You can use the Patch rule for that

Reply:
Oh cool! Does the patch rule work if the patch is an identically named esp replacer? I was worried that would be an issue, hence making it a note rather than a patch rule

Reply:
Oh I See, 😀 Nevermind then, I have no idea if it works